builtin/providers/aws/resource_aws_api_gateway_method.go

```diff
@@ -89,6 +94,7 @@ func resourceAwsApiGatewayMethodCreate(d *schema.ResourceData, meta interface{})
 		// TODO reimplement once [GH-2143](https://github.com/hashicorp/terraform/issues/2143) has been implemented
 		RequestParameters: aws.BoolMap(parameters),
 		ApiKeyRequired:    aws.Bool(d.Get("api_key_required").(bool)),
+		AuthorizerId:      aws.String(d.Get("authorizer_id").(string)),
```

Comment:
I think we'll need to update the other functions too so that user can _change_ the `authorizer_id` and Terraform can read it back if it's changed outside of Terraform.

i.e.

`resourceAwsApiGatewayMethodRead` will need `d.Set("authorizer_id", out.AuthorizerId)` - I noticed we're missing some other fields there - if you want to fix that too, that's great, but we can fix that in a separate PR.

and `resourceAwsApiGatewayMethodUpdate` will need `if d.HasChange("authorizer_id") {` block.